css/responsive/desktop.styl

```diff
@@ -6,6 +6,7 @@ $gutter = 32px
   
 .col-l
   padding: 0 ($gutter / 2)
+  margin-bottom: ($gutter / 2)
```

Comment:
wait why does this class still exist. any values here should just override `.col`